sklearn/linear_model/_ridge.py

```diff
@@ -2560,10 +2560,14 @@ class RidgeCV(MultiOutputMixin, RegressorMixin, _BaseRidgeCV):
         (i.e. data is expected to be centered).
 
     scoring : str, callable, default=None
-        A string (see :ref:`scoring_parameter`) or a scorer callable object /
-        function with signature ``scorer(estimator, X, y)``. If None, the
-        negative mean squared error if cv is 'auto' or None (i.e. when using
-        leave-one-out cross-validation), and r2 score otherwise.
+        The scoring method to use for cross-validation. Options:
+
+        - str: see :ref:`scoring_string_names` for options.
+        - callable: a scorer callable object (e.g., function) with signature
+          ``scorer(estimator, X, y)``. See :ref:`scoring_callable` for details.
+        - `None`: negative :ref:`mean squared error <mean_squared_error>` if cv is
+          None (i.e. when using leave-one-out cross-validation), and
+          :ref:`coefficient of determination <r2_score>` (:math:`R^2`) otherwise.
```

Comment:
@StefanieSenger I get what you meant when you said Ridge is confusing. `cv` can no longer be 'auto' (I assume thats been deprecated) so removing that helps.

Essentially when using `cv=None`, we use the default efficient LOO cv, in which case negative mean squared error is used. R2 is used for all other values of `cv`. Now that I've figured out what this means, it makes sense as it is and I can't think of a better way to put it without making it too long. Suggestions welcome.

Reply:
Yeah no idea where 'auto' came from.

>  if the gcv_mode is meant with it.

I don't think so because I think mean squared error is the default no matter what gcv mode is:

https://github.com/scikit-learn/scikit-learn/blob/caaa1f52a0632294bf951a9283d015f7b5dd5dd5/sklearn/linear_model/_ridge.py#L2174-L2175

also 'auto' could be either 'svd' or 'eigen' so it doesn't make sense to say score is *only* a specific thing when `gcv_mode` is auto.